packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -249,20 +249,17 @@ export const RemoteServerFactory = {
     },
 
     /**
-     * Passing the requestId from apiGateway event to locals
+     * Passing the correlation Id from MRT to locals
      * @private
      */
     _setRequestId(app) {
         app.use((req, res, next) => {
-            if (!req.headers['x-apigateway-event']) {
```

Comment:
just double checking...

The header `x-apigateway-event` will still be available from MRT right? We need to make sure the old sdk versions continue to work on MRT.

Reply:
What did we use the `x-apigateway-event` header for before? 

Also, I see that we reference `x-apigateway-event` in a few other places, such as in tests. Do we need to replace those with `x-correlation-id` as well?

Reply:
I believe that APIG is actually expected to be removed next release. @drewzboto could you confirm this?

Reply:
This is a really important discussion.

We need to consider sites that are already live and that haven't been redeployed and sites running older versions of the PWA Kit.

In this case, we're in good shape because this code "guards"  against the absence of the both the new and the old headers, but we might consider continuing to send the "old" header with our own generated value to "backport" the feature to folks using earlier versions of the PWA Kit.

Reply:
Gotcha. @johnboxall Would it be okay to set `res.locals.requestId` to the `x-apigateway-event` header if `x-correlation-id` isn't available? Or is this undesirable as then we don't know which value `res.locals.requestId` is representing?